native/candlex/src/tensors.rs

```diff
@@ -390,6 +390,36 @@ pub fn conv2d(
     )?))
 }
 
+#[rustler::nif(schedule = "DirtyCpu")]
+pub fn max_pool2d(
+    tensor: ExTensor,
+    dimensions: Vec<usize>,
```

Comment:
Same here about argument name `dims`

Reply:
okay i'll update the this PR and create an other PR for window_mean in the same vein tomorrow. Thx for your comments !

Reply:
> okay i'll update this PR and create an other PR for window_mean in the same vein tomorrow. Thx for your comments !

Just realized `Nx.window_mean` doesn't need a `window_mean` implemented in the backend. It uses the backend `window_sum`, so we actually need `Candlex.Backend.window_sum` to support both `Nx.window_sum` and `Nx.window_mean`.